R/utilities-configuration-plan.R

```diff
@@ -26,21 +26,38 @@ ConfigurationScales <- list(
 )
 
 #' @title ConfigurationShapes
-#' @description List mapping configuration plan symbols to tlf shapes
+#' @description List mapping configuration plan symbols to tlf shapes.
+#' Note that the names only include lower case as the mapping 
+#' used in `tlfShape()` internally translate configuration plan field with `tolower()` function.
 #' @import tlf
 #' @keywords internal
 ConfigurationShapes <- list(
   none = tlf::Shapes$blank,
-  circle = tlf::Shapes$hollowCircle,
-  dot = tlf::Shapes$dot,
-  square = tlf::Shapes$hollowSquare,
-  diamond = tlf::Shapes$hollowDiamond,
-  asterisk = tlf::Shapes$asterisk,
+  circle = tlf::Shapes$circle,
   cross = tlf::Shapes$cross,
+  diamond = tlf::Shapes$diamond,
+  dot = tlf::Shapes$circle,
```

Comment:
let's remove "dot".
It is not available in PK-Sim; in case it was used in a qualification plan - we can change it to another symbol there.

Reply:
(I don't think "dot" was ever used in a quali plan anyways)